packages/devtools-launchpad/src/components/LandingPage.js

```diff
@@ -134,30 +136,36 @@ const LandingPage = React.createClass({
     const isConnected = name == "Firefox"
       ? this.state.firefoxConnected
       : this.state.chromeConnected;
-    const launchButton = isConnected ? null : dom.input({
+
+    const connectedStateText = isNodeSelected ?
+      null : `Please open a tab in ${name}`;
+
+    const launchButton = isConnected ? connectedStateText : dom.input({
       type: "button",
       value: `Launch ${selectedPane}`,
       onClick: () => launchBrowser(selectedPane)
     });
+
+    const targetsContent = currentTargetsExist ? dom.header({},
```

Comment:
It'd be nice to add storybook stories `yarn storybook` for these cases too. Not necessary though :)